exporter/clickhouseexporter/README.md

```diff
@@ -375,6 +408,194 @@ WHERE TraceId != ''
 GROUP BY TraceId;
 ```
 
+### Metrics
+
+#### Gauge
+
+```clickhouse
+CREATE TABLE IF NOT EXISTS otel.otel_metrics_gauge (
+    ResourceAttributes Map(LowCardinality(String), String) CODEC(ZSTD(1)),
+    ResourceSchemaUrl String CODEC(ZSTD(1)),
+    ScopeName String CODEC(ZSTD(1)),
+    ScopeVersion String CODEC(ZSTD(1)),
+    ScopeAttributes Map(LowCardinality(String), String) CODEC(ZSTD(1)),
+    ScopeDroppedAttrCount UInt32 CODEC(ZSTD(1)),
+    ScopeSchemaUrl String CODEC(ZSTD(1)),
+    MetricName String CODEC(ZSTD(1)),
+    MetricDescription String CODEC(ZSTD(1)),
+    MetricUnit String CODEC(ZSTD(1)),
+    Attributes Map(LowCardinality(String), String) CODEC(ZSTD(1)),
+    StartTimeUnix DateTime64(9) CODEC(Delta, ZSTD(1)),
+    TimeUnix DateTime64(9) CODEC(Delta, ZSTD(1)),
+    ValueAsDouble Float64 CODEC(ZSTD(1)),
+    ValueAsInt Int64 CODEC(ZSTD(1)),
```

Comment:
Why would you want to maintain two types at the DB storage for value? Why not just make float64 for all the values? How would one know which one to use when querying?

Reply:
done